Makefile

```
@@ -255,7 +255,7 @@ index.html: blog/announcements-rss.xml blog/index.html
 community/commercial-support.html: community/commercial-support.html.in
 
 community/commercial-support.html.in: community/commercial-support.toml
```

Copying my comment from https://github.com/NixOS/nixos-homepage/pull/1052#discussion_r1193009005:

> It should be possible to just have
> 
> ```suggestion
> community/commercial-support.html: community/commercial-support.toml
> ```
> 
> and remove the `.in` file from git.